Automattic/jetpack
# Add a 'copy' button for emails on the jetpack forms dashboard

## Proposed changes:

This PR adds a 'Copy' button for the email on the response tab in Jetpack Forms dashboard. The button will appear when hovering over the line and will copy the email address to the clipboard.

![Screenshot 2023-04-24 at 09 47 25](https://user-images.githubusercontent.com/8056203/233932899-30f1e9f3-473b-4cb1-9475-bd66cf2c9f23.png)

### Other information:


## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






- Go to `admin.php?page=jetpack-forms`. Make sure you already have some responses in there.
- Hover over the email address in the header of the response tab.
- You should see a 'Copy' button next to the email.
- Clicking the button should copy the email address to the clipboard.
